docs/user-guide/admin-cli.md

```diff
@@ -154,22 +215,35 @@ optional arguments:
 
 ### Update
 Update existing vmgroup. This command allows to update "Description" and "Default_datastore" fields, or rename an existing vmgroup.
+Valid value for "Default_datastore" includes the name of valid datastores in the ESX host or a special value "__VM_DS".
```

Comment:
"Default_datastore" is either a  valid datastore name or a special value "__VM_DS".